games-util/springlobby/springlobby-0.264.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+WX_GTK_VER="3.0"
+inherit cmake-utils eutils flag-o-matic wxwidgets
```

Comment:
You aren't using half of those eclasses.

Reply:
fixed